assignment1.rb

```diff
@@ -579,16 +572,12 @@ def swap(a, b)
 # 25. Program to Count the Number of Each Vowel
 
 def count_each_vowel(str)
-    puts "a = #{str.count('a')}"
-    puts "e = #{str.count('e')}"
-    puts "i = #{str.count('i')}"
-    puts "o = #{str.count('o')}"
-    puts "u = #{str.count('u')}"
+    return str.count('a'), str.count('e'), str.count('i'), str.count('o'), str.count('u')
 end
 
```

Comment:
You can use array with include?

%w[a e i o u].include?(str)

Reply:
I tried this method and some other similar to it but none of them are giving desirable output

Reply:
Pls ignore. I got confused. This will not work since we need the count for each vowel. This would work when we need the count for all.